chain/epoch-manager/src/adapter.rs

```diff
@@ -122,7 +122,7 @@ pub trait EpochManagerAdapter: Send + Sync {
         &self,
```

Comment:
Should we update the comment to reflect the new return type? I won't suggest changing the method name unless somebody sees any value in doing so

Reply:
Also valid for other modified methods in this file

Reply:
I'll add a todo to follow up on that.